src/main/kotlin/com/terraformation/backend/accelerator/db/SubmissionStore.kt

```diff
@@ -14,15 +14,15 @@ import com.terraformation.backend.db.accelerator.tables.references.SUBMISSION_DO
 import com.terraformation.backend.db.asNonNullable
 import com.terraformation.backend.db.default_schema.ProjectId
 import jakarta.inject.Named
-import java.time.InstantSource
+import java.time.Clock
 import org.jooq.Condition
 import org.jooq.DSLContext
 import org.jooq.impl.DSL
 import org.springframework.context.ApplicationEventPublisher
 
 @Named
 class SubmissionStore(
-    private val clock: InstantSource,
+    private val clock: Clock,
```

Comment:
I've been using `InstantSource` rather than `Clock` just to keep the interfaces minimal. `Clock` inherits from `InstantSource` and adds a bunch of time-zone-related functionality we usually don't need.